assignment-2/power.md

```diff
@@ -1,3 +1,127 @@
+
+
+### Who was in control?
+
+######CMS
+CMS (Centers for Medicare and Medicaid Services) was responsible for the overall coordination and planning of the project. But it did fail according to reports of the office of inspector general.
+
+
+According to one report [1] there were inadequacies in contract planning and procurement. The report states: <i>“When awarding the Federal Marketplace contracts, CMS did not always meet contracting requirements. For example, CMS did not develop an overarching acquisition strategy for the Federal Marketplace or perform all required oversight activities. Moreover, for a project of this size and importance, CMS missed opportunities to leverage all available acquisition planning tools and contracting approaches to identify and mitigate risks. Specifically, CMS did not exercise the option to plan for a lead systems integrator to coordinate all contractors’ efforts prior to the launch of the Federal Marketplace. The complexity of the Federal Marketplace underscored the need for CMS to select the most qualified contractors. However, CMS did not perform thorough reviews of contractor past performance when awarding two key contracts. CMS also made contracting decisions that may have limited the number of acceptable proposals for much of the key Federal Marketplace work. In addition, CMS selected contract types that placed the risk of cost increases for this work solely on the Government.“ </i>. 
+
+According to another report of the office of inspector general [3] CMS did not alway manage and oversee the contractor performance. The main problem in this case is the lack of control on the contract deliverables.
+
+######Obama administration
+Also the Obama administration, did not make this project easy, it kept modifying regulations and policies of the Affordable Care Act [1], which inherently meant that the scope of the HealthCare.gov website kept changing too. Which probably has caused some hardship with the changes trickling down to all the contractors, but there is a lack of documentation on how the changes were documented and implemented.
+
+
+#######Culture: Alienative, Calculative vs. Moral involvement
+Etzioni [6] [7] developed the compliance theory. He classifies organizations by the type of power they use to direct the behavior of their members and the type of involvement of the participants. He identified three types of organizational power: coercive, utilitarian, and normative, and relates these respectively to three types of involvement: alienative, calculative, and moral.  
+
+
+(still need to written coupling between Etzioni theory and culture in Washington DC)
+
+According to Robert Kennedy [8] a Google employee that worked in is free time in Washington DC at the HealthCare.gov project people were afraid of making mistakes, because they could be fired for it. This way people did not raise any mistakes that they made and therefore the problems that result from these mistakes couldn’t be easily resolved.
```

Comment:
- "According to Robert Kennedy [8] a Google employee that worked in is free time in Washington DC at the HealthCare.gov project people were afraid of making mistakes, because they could be fired for it." Misplaced comma. Move it between 'project people'.
- the problems that resultED